lib/request.js

```diff
@@ -103,7 +104,10 @@ const ajax = ({ method, url, headers, body = null }, callback) => {
   xhr.onload = () => {
     $html.classList.remove('swap-is-loading');
     if (xhr.status !== 200) {
-      alert('Error: ' + xhr.status);
+      if (method === 'get') {
```

Comment:
@shanebo I update this to only run on `get` requests. This is because we are expecting form submits (which I'm assuming are going to be `post` methods) to not render the error page. See this existing spec: https://github.com/shanebo/swap/blob/0.1.0/cypress/integration/pane_spec.js#L193-L203. LMK if there is a different way you think we should be handling this.